### PR TITLE
feat: add link to support page in Innoactive menu

### PR DESCRIPTION
**Pull request by niklasthiem1**
### Description

ia

Added a menu entry which enables the user to report issues via Keycloak Auth and subsequently via Jira Service Desk

### Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by niklasthiem1** [comment]
@Gusinuhe whats wrong?

**Comment by Gusinuhe** [comment]
> @Gusinuhe whats wrong?

It looked like the description was left empty 😅
But now it is updated 👍

**Comment by niklasthiem1** [comment]
Ah okay you are right, sorry i put it on **draft** meanwhile. 

Btw is there a certain convention for the menu item priority? 

E.g: 
image

**Comment by Gusinuhe** [comment]
> Ah okay you are right, sorry i put it on **draft** meanwhile.
> 
> Btw is there a certain convention for the menu item priority?
> 
> E.g:
> image

Maybe 🤔
we must summon @SimonTheSourcerer!

**Comment by SimonTheSourcerer** [comment]
No Convention... but there are some rules by unity when and how lines a drawn and stuff. But dont ask my how exactly

**Comment by SimonTheSourcerer** [comment]
I dont see any problems, do we just want to add it?

**Review by SimonTheSourcerer — APPROVED** [review]

**Review by Gusinuhe — APPROVED** [review]

File: Editor/UI/CreatorMenu/SupportMenuEntry.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    {
+        /// <summary>
+        /// Allows to open the URL to Innoactive's Jira Servicedesk.
+        /// </summary>
+        [MenuItem("Innoactive/Support", false, 80)]
+        private static void OpenSupportPage()
+        {
+            Application.OpenURL("https://innoactive.io/creator/support");
+        }
+
```
Comment (tomwim) [inline Editor/UI/CreatorMenu/SupportMenuEntry.cs:16]: EMPTY SPACE!!! 😱

**Review by tomwim — APPROVED** [review]

**Pull request merged**